provider/instance_client.go

```diff
@@ -104,7 +111,7 @@ func (ic *InstanceClient) saveProfileScript() error {
 }
 
 func (ic *InstanceClient) configureService() error {
-	if !ic.data.System.ServiceEnabled || ic.data.Client.Type == "local" {
+	if !ic.data.System.ServiceEnabled {
```

Comment:
I don't remeber such bool `serviceEnabled` in TF provider... why have you introduced it? these providers should not be so different

Reply:
system service is must have; we want to maintain only the single and correct way of setting up AEM instances... maintaining variations will increase complexity

Reply:
hmm if serviceEnabled = false then runServiceCommand("start") need be replaced by "sh aemw instance launch" command and the similar thing for stopping

Reply:
for non-admin usages like on local/dev machine it would be fine